server/protocols/jmap/src/main/java/org/apache/james/jmap/utils/SystemMailboxesProviderImpl.java

```diff
@@ -41,7 +42,7 @@
     private final MailboxManager mailboxManager;
 
     @Inject
-    @VisibleForTesting SystemMailboxesProviderImpl(MailboxManager mailboxManager) {
+    public SystemMailboxesProviderImpl(MailboxManager mailboxManager) {
```

Comment:
I tried but later on, the constructor is used at PostDequeueDecoratorTest so it cannot be private. 
On PostDequeueDecoratorTest, I tried to make a mock for SystemMailboxesProviderImpl but it will be complicated after that.
How can I do in this case?